_topic_maps/_topic_map.yml

```diff
@@ -183,13 +183,18 @@ Distros: openshift-acs
 Topics:
 - Name: Viewing the dashboard
   File: view-dashboard
-- Name: Managing compliance
-  File: manage-compliance
 - Name: Using the Compliance Operator
   Dir: manage-compliance-operator
   Topics:
   - Name: Using Compliance Operator with RHACS
     File: compliance-operator-rhacs
+- Name: Managing Compliance
+  Dir: manage-compliance
+  Topics:
+  - Name: Managing compliance (1.0)
+    File: managing-compliance-10
+  - Name: Managing compliance (2.0)
+    File: managing-compliance-20
```

Comment:
Can we do without the brackets, for example, Managing compliance 1.0, we still need to use the version numbers to differentiate between the fetaures.
We also have similar versioning used in Vulnerability management: 
https://71171--docspreview.netlify.app/openshift-acs/latest/operating/manage-vulnerabilities/vulnerability-management#vulnerability-management20-view-workload-cve_acs-operating-manage-vulnerabilities

@kcarmichael08 Please chime in on this one?

Reply:
If we follow the style guides to the letter, we need the parentheses, unfortunately, since they're menu items. One thing that we could do is remove the parenthesis in headings and only use them when we're referring to the actual menu items in an action (e.g., "Click *Compliance (2.0)*").

Reply:
That would make it somewhat inconsistent with how we've done Vuln Management 1.0 & 2.0, which does have the parentheses in headings. Unfortunately it looks like for the foreseeable future, both Compliance and Vuln Management are going to have both versions and parentheses in the GUI. (at least for 4.4 & possibly 4.5)